src/Controls/src/Core/TabbedPage/TabbedPage.Windows.cs

```diff
@@ -48,6 +49,29 @@ FrameworkElement CreatePlatformView()
 			return _navigationFrame;
 		}
 
+		private void OnPagePropertyChanged(object? sender, PropertyChangedEventArgs e)
+		{
+			if (e.PropertyName == Page.IconImageSourceProperty.PropertyName)
+			{
+				if (sender is Page page)
```

Comment:
Can this just fire a mapper change on `MapItemsSource" and that'll handle it?

Handler?.UpdateValue("ItemSource") ? 

MapItemsSource already has code that loops over the navigation items and syncs all the colors and icons

Reply:
I checked this by using `Handler?.UpdateValue("ItemSource")` in PropertyChanged. The icon color changed dynamically, but the title color reset to black because MapItemSource also handled the title color.
 
When changing the IconImageSource color dynamically, our fix worked properly, preventing unnecessary changes to the title color and other properties.
 
Please let me know if you have any other concerns @PureWeen